packages/core/src/actions/account/wallet/create-remote-account.ts

```diff
@@ -0,0 +1,60 @@
+import { IbcClientTypes, ManagerClient } from '../../../codegen/abstract'
+import { WithArgsAndCosmWasmSignOptions } from '../../../types/with-args'
+import { executeIbcAction } from './execute-ibc-action'
+import { BaseWalletParameters } from './types'
+
+export type CreateRemoteAccountParameters = Omit<
+  WithArgsAndCosmWasmSignOptions<
+    BaseWalletParameters &
+      Omit<
+        Extract<IbcClientTypes.ExecuteMsg, { register: unknown }>['register'],
+        'host_chain'
+      > & {
+        hostChainName: string
```

Comment:
We might want to call this `remoteChainName`

Reply:
better to stick with the names that we have in the contract to avoid confusion.

we can provide this one as an alias maybe.